### PR TITLE
George style read

### PR DESCRIPTION
**Pull request by georgeSoton**

### CODE REVIEW

File: data_processing.tex [hunk 1]
```diff
@@ -5,27 +5,29 @@ \section{Data Processing}
...
+    (or truncating) sentences to a known length and
+    windowing. Several windows can be gained from each
     sentence: augmenting the dataset.
 
-    For example, the sentence \textit{``This process, however, afforded me no
-      means of ascertaining the dimensions of my dungeon [...]''} would produce windows
-    such as \texttt{[this process , however ,], [process ,
-      however , afforded], [, however , afforded me], ...}
-
     Two encodings for the class labels have been experimented with: one hot
```
Comment (tblah) [inline data_processing.tex]: lines 28 and 29 need to be removed as they are now superseeded by the new author representation subsection

File: ml_report.tex [hunk 2]
```diff
@@ -15,7 +15,7 @@
 \usepackage{tikz}
 \MakeOuterQuote{"}
 
-\usepackage[backend=biber,sorting=none, urldate=edtf, date=edtf]{biblatex}
+\usepackage[backend=bibtex,sorting=none, urldate=edtf, date=edtf]{biblatex}
```
Comment (tblah) [inline ml_report.tex]: Are we sure that bibtex supports everything? It is an older version of biblatex
https://tex.stackexchange.com/questions/25701/bibtex-vs-biber-and-biblatex-vs-natbib#25702

**Review by tblah — APPROVED** [review]

File: ml_report.tex (hunk 2 above)
Comment (Adimote) [reply ml_report.tex]: also this breaks the CI build

Comment (Adimote) [reply ml_report.tex]: it also breaks compiling on my machine, because I use the make file

Comment (tblah) [reply ml_report.tex]: If we are to keep this then we need to change the makefile not to run biber (that will fix the CI error)

**Pull request merged**